stream_chat/types/rate_limit.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import asdict, dataclass
+from datetime import datetime
+from typing import Dict
+
+
+@dataclass(frozen=True)
+class RateLimitInfo:
+    limit: int
+    remaining: int
+    reset: datetime
+
+    def as_dict(self) -> Dict[str, int]:
+        return asdict(self)
```

Comment:
where is this used?

Reply:
it was used before, but I just removed the code that used it. deleting it now.